store/snap_remote_repo.go

```diff
@@ -241,6 +245,21 @@ func (s *SnapUbuntuStoreRepository) configureStoreReq(req *http.Request, accept
 	s.applyUbuntuStoreHeaders(req, accept)
 }
 
+// read all the available metadata from the store response and cache
+func (s *SnapUbuntuStoreRepository) readStoreResponse(resp *http.Response) {
```

Comment:
"reading a response" has a specific meaning which isn't intended here.

Let's please name this `checkStoreResponse`.